src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -179,9 +179,24 @@ LRESULT IslandWindow::_OnSizing(const WPARAM wParam, const LPARAM lParam)
     // If this fails, we'll use the default of 96.
     GetDpiForMonitor(hmon, MDT_EFFECTIVE_DPI, &dpix, &dpiy);
 
+    // The minimum client width that you can resize to.
+    // This lets the tabs fit.
+    const auto minClientWidth = 460;

```

Comment:
Can you make this a global static constexpr til::size?

Reply:
then you can use the size scaling operators.

Reply:
Adding a global static constexpr til::size.

Can't use the scaling operators though :( scaling is a float and you can't have a til::point/size be a float.